patches/chrome-browser-chrome_content_browser_client.cc.patch

```diff
@@ -1,13 +0,0 @@
-diff --git a/chrome/browser/chrome_content_browser_client.cc b/chrome/browser/chrome_content_browser_client.cc
-index 0b29bc11f07c95f6048c9a35150925ad16a08586..44cab1c7f2897fb0191b33653d3cae945234d79f 100644
---- a/chrome/browser/chrome_content_browser_client.cc
-+++ b/chrome/browser/chrome_content_browser_client.cc
-@@ -3376,7 +3376,7 @@ void ChromeContentBrowserClient::OverrideWebkitPrefs(
-                                  : blink::ForcedColors::kNone;
- #endif  // !defined(OS_MACOSX)
- 
--  switch (native_theme->GetPreferredColorScheme()) {
```

Comment:
I think `GetPreferredColorScheme` will return proper scheme when we set our dark mode and re-calculateded color scheme.